package.json

```diff
@@ -11,6 +11,7 @@
     "mousetrap": "1.6.0",
     "ms": "0.7.1",
     "object-values": "1.0.0",
+    "php-escape-shell": "^1.0.0",
```

Comment:
dependencies are pinned because there are currently no tests, so a package *could* break hyperterm without us knowing
https://github.com/zeit/hyperterm/pull/513